.opspec/release/op.yml

```diff
@@ -40,22 +40,25 @@ inputs:
       description: version of opctl being released
 run:
   serial:
+    - op:
+        ref: $(../changelog/get-latest-version)
+        outputs:
+          latestVersion:
     - op:
```

Comment:
do you think we need to add a condition here to skip if latestVersion == currentVersion ?

Reply:
right now it will run every time (and re-publish over existing) right?

Reply:
good call - I've updated the release op to first make sure a tag doesn't exist in GHCR

Reply:
Though wouldn't the same problem have existed in the previous implementation? I don't think the docker registry is immutable

Reply:
previously releases were manual so it was up to the releaser to ensure.